modules/compiler/vecz/source/transform/packetizer.cpp

```diff
@@ -803,6 +819,38 @@ PacketRange Packetizer::createPacket(Value *V, unsigned width) {
   return Result(*this, V, &info).createPacket(width);
 }
 
+Packetizer::Result Packetizer::Impl::getPacketizationResult(
+    Instruction *I, const SmallVectorImpl<Value *> &Packet, bool UpdateStats) {
+  if (Packet.empty()) {
+    return Result(*this);
+  }
+  auto PacketWidth = Packet.size();
+
+  if (PacketWidth == 1) {
```

Comment:
could add a comment here to explain why PacketWidth==1 is different?

Reply:
Done, thanks